test/pubsub/subscription.js

```diff
@@ -107,6 +108,10 @@ describe('Subscription', function() {
     it('should not be paused', function() {
       assert.strictEqual(subscription.paused, false);
     });
+
+    it('should create an iam object', function() {
```

Comment:
I know this will be annoying, but we have to see that IAM is called with the correct arguments: https://github.com/callmehiphop/gcloud-node/blob/0609a6cad7f333c0051b8e885d899f9124e59a20/lib/pubsub/subscription.js#L179

That unfortunately means using mockery to create a fake IAM.